src/components/toolbar/Toolbar.tsx

```diff
@@ -39,6 +39,7 @@ export interface ToolbarItemProps extends ToolbarBaseProps {
   active?: boolean;
   onClick?: (item: ToolbarItemProps) => void;
   className?: string;
+  tooltip?: boolean;
```

Comment:
I approved the PR a bit too quickly...

Can you make this a `noTooltip`. We always try to name props so that the default is `false`
- It's more intuitive to use since not specifying it is the same as setting it to `false`
- It's nicer syntaxically to do `<Toolbar.Item noTooltip />` `<Toolbar.Item tooltip={false} />`

Reply:
fixed, thank you for the explication.